### PR TITLE
DF-178 Fix support of Firefox and IE styles

### PR DESCRIPTION
**Pull request by varusakov**

### CODE REVIEW

File: src/main/webapp/app/layouts/navbar/navbar.scss [hunk 1]
```diff
@@ -71,6 +71,6 @@ Style guide customization
 @media screen and (min-width: 767px){
     .navbar-nav .dropdown-menu--account.show {
         position: absolute;
-        width: fit-content;
+        display: table;
```
Comment (cjpsycos) [inline src/main/webapp/app/layouts/navbar/navbar.scss]: I don't think this is the right value. did you try display: inline-block?

Comment (varusakov) [reply src/main/webapp/app/layouts/navbar/navbar.scss]: `display: inline-block` doesn't work (tested in FF, Chrome, IE11 and Safari).

File: src/main/webapp/app/layouts/navbar/navbar.scss [hunk 2]
```diff
@@ -71,6 +71,7 @@ Style guide customization
 @media screen and (min-width: 767px){
     .navbar-nav .dropdown-menu--account.show {
         position: absolute;
-        width: fit-content;
+        display: inline-block;
```
[the remark this answers is not in the document]
Comment (ganqqwerty) [reply src/main/webapp/app/layouts/navbar/navbar.scss]: What’s the intention here?

[the remark this answers is not in the document]
Comment (birom) [reply src/main/webapp/app/layouts/navbar/navbar.scss]: To fix this:
selection_108

**Pull request merged**